mlrun/model_monitoring/model_monitoring_batch.py

```diff
@@ -468,6 +468,13 @@ def calculate_inputs_statistics(
                 counts.tolist(),
                 bins.tolist(),
             ]
+        else:
+            # Comply with the other common features' histogram length
+            mlrun.common.model_monitoring.helpers.pad_hist(
+                mlrun.common.model_monitoring.helpers.Histogram(
+                    inputs_statistics[feature]["hist"]
+                )
+            )
```

Comment:
Can you please explain a bit why if a certain feature doesn't exist in `sample_set_statistics` we need to apply the pad histogram function on it? 
The `inputs_statistics` in this case also includes the `timestamp` column which has no histogram at all and therefore you'll get the following err:
```
 File "/usr/local/lib/python3.9/site-packages/mlrun/package/__init__.py", line 140, in wrapper
    func_outputs = func(*args, **kwargs)
  File "model_monitoring_batch.py", line 996, in handler
    raise batch_processor.exception
  File "model_monitoring_batch.py", line 742, in update_drift_metrics
    current_stats = calculate_inputs_statistics(
  File "model_monitoring_batch.py", line 475, in calculate_inputs_statistics
    inputs_statistics[feature]["hist"]
KeyError: 'hist'
```

Reply:
> Can you please explain a bit why if a certain feature doesn't exist in `sample_set_statistics` we need to apply the pad histogram function on it?

Because later on all the histograms are passed down to a shared data-frame (as we discussed orally)

> The `inputs_statistics` in this case also includes the `timestamp` column which has no histogram at all and therefore you'll get the following err:

Fixed - checking for the key existence first now